BuildConfigGen/Debugging/DebugConfigGenerator.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace BuildConfigGen.Debugging
+{
+    internal interface IDebugConfigGenerator
+    {
+        void WriteTypescriptConfig(string taskOutput);
+
+        void AddForTask(string taskConfigPath);
```

Comment:
Shouldn't we have RemoveForTask as well?

Reply:
No, why? This flow won't ever remove configs, it only updates the existing ones. You can clean the configs simply by going to vscode launch.json or through its UI if there are too many which you don't want. We wouldn't know what to clean here, or whether it's "ours" or "yours". We don't want to clean something you need.